File: test/fixtures/r/post_escaped_single_quotes_in_double_quotes.r
require(httr)

headers = c(
  `Content-Type` = 'application/x-www-form-urlencoded'
)

data = list(
  `foo` = '\\\'bar\\\''
)

res <- httr::POST(url = 'http://localhost:28139/', httr::add_headers(.headers=headers), body = data)

File: test/fixtures/dart/post_escaped_single_quotes_in_double_quotes.dart
import 'package:http/http.dart' as http;

void main() async {
  final headers = {
    'Content-Type': 'application/x-www-form-urlencoded',
  };

  final data = "foo=\\'bar\\'";

  final url = Uri.parse('http://localhost:28139/');

  final res = await http.post(url, headers: headers, body: data);
  final status = res.statusCode;
  if (status != 200) throw Exception('http.post error: statusCode= $status');

  print(res.body);
}
